### DISCUSSION TITLE
Red Hat Enterprise Linux 9 support

### DISCUSSION THREAD

**Discussion by hisa-tanaka** [fluent/fluent-bit#5310] [General]
Is there any timeline or a roadmap for Red Hat Enterprise Linux 9 support from Fluent Bit?

Since the beta was released back in November last year, I would like to see if there is an ongoing discussion around it.

Thank you.

**Comment by patrick-stephens** [comment]
See https://github.com/fluent/fluent-bit/issues/5478
